.doc_gen/metadata/cloudfront_metadata.yaml

```diff
@@ -79,3 +79,130 @@ cloudfront_UpdateDistribution:
                 - python.example_code.cloudfront.UpdateDistribution
   services:
     cloudfront: {UpdateDistribution}
+cloudfront_CreateDistribution:
+  title: Create a &CF; distribution using an &AWS; SDK
+  title_abbrev: Create a distribution
+  synopsis: create a &CF; distribution.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/cloudfront
+          sdkguide:
+          excerpts:
+            - description: >-
+                The following example uses an Amazon S3 bucket as a content origin. 
+                <para>After creating the distribution, 
+                the code creates a <ulink url="https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/cloudfront/waiters/CloudFrontWaiter.html">CloudFrontWaiter</ulink> 
+                to wait until the distribution is deployed before returning the distribution</para>.
```

Comment:
`distribution</para>.  --> distribution.</para>`

Reply:
I had to put the period outside of the </para> tag or the file would not validate, since a period is required at the end of a sentence.